scripts/empty_files/antarctica-icesheet-outlines-3857.zip.lastmod

```diff
@@ -0,0 +1 @@
+Thu 01 Jan 1970 12:00:00 AM UTC
```

Comment:
This (and other) files has a different format than external_data.last_modified, causing an issue. TODO: Change these to `Thu, 01 Jan 1970 00:00:00 GMT` with no newline.